mkdocs.yml

```diff
@@ -45,6 +45,7 @@ nav:
                          - 'Packaging and Delivering Software with IPS': dev/pdf/ips-dev-guide.pdf
                          - 'Using distribution constructor': dev/distribution-constructor.md
                          - 'Existing tasks': dev/existing-tasks.md
+                - 'Squeak on OpenIndiana': handbook/community/squeak.md
```

Comment:
Add link to handbook/community.md itself, like it's done with Oracle documentation.